docs/en/mavlink/protocols.md

```diff
@@ -0,0 +1,53 @@
+# MAVLink Microservices (Protocols)
+
+MAVLink "microservices" are a protocols that use multiple messages exchanged between components to communicate more complicated information.
+For example, the [Command Protocol](https://mavlink.io/en/services/command.html) provides an efficient mechanism for packaging a command in a (particular) message and receiving acknowledgement of the command in another message.
+
+MAVLink microservices are documented the [MAVLink Guide](https://mavlink.io/en/services/) (this is not exhaustive: not all messages are grouped into protocols and not all protocols are documented).
+
+This section lists the services known to be supported/not supported by PX4 in this version.
+
+## Supported Microservices
```

Comment:
This is the new doc. Are all these protocols supported "to your knowledge". I know there are some clarifications to add, such as more detail on gimbal integration, and the fact that we don't "directly" support mavlink cameras.